beta-src/src/components/ui/WDCountdownPill.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { Avatar, Chip, useTheme } from "@mui/material";
+import BlackCountdownIcon from "../../assets/png/icn_phase_countdown_black.png";
+import RedCountdownIcon from "../../assets/png/icn_phase_countdown_red.png";
+import parseSeconds from "../../utils/parseSeconds";
+import { ParsedTime } from "../../interfaces/ParsedTime";
+import formatTime from "../../utils/formatTime";
+
+interface WDCountdownPillProps {
+  endTime: number;
+  phaseTime: number;
+}
+
+const WDCountdownPill: React.FC<WDCountdownPillProps> = function ({
+  endTime,
+  phaseTime,
+}) {
+  const theme = useTheme();
+  const endTimeInMilliSeconds = endTime * 1000;
```

Comment:
let's make this a const outside this component. the reason is so that 1000 doesn't accidentally get changed when someone edits this code in the future. so above the component make a const MS = 1000;